test-api-generator/citrus-openapi-codegen-maven-plugin/src/test/resources/TestApiGeneratorMojoIntegrationTest/pom-no-model-config.xml

```diff
@@ -0,0 +1,37 @@
+<project
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>multi-config</artifactId>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>citrus-openapi-codegen-maven-plugin</artifactId>
+                <configuration>
+                    <apis>
+                        <api>
+                            <prefix>NoModel</prefix>
+                            <source>api/test-api.yml</source>
+                            <apiConfigOptions>
+                                <!-- configure no model generation -->
+                                <generateModels></generateModels>
```

Comment:
that looks strange to my eye.. but I know it's exactly what you wanted to test. anyway, I think it should be either `true` or `false` - imo.